### PR TITLE
Update teams-for-vdi.md

### PR DESCRIPTION
**Pull request by fernandoklurfan**
updated the system requirements

### CODE REVIEW

File: Teams/teams-for-vdi.md [hunk 1]
```diff
@@ -115,7 +115,9 @@ The following is the recommended minimum VM configuration.
 |---------|---------|---------|
 |vCPU   |    2 cores     |  4, 6, or 8 cores<br>It's important to understand the underlying non-uniform memory access (NUMA) configuration and configure your VMs accordingly.     |
 |RAM     |   4 GB      | 512 MB to 1 GB per user        |
-|Storage    | 8 GB        | 40 GB to 60 GB        |
+|Storage    | 32 GB        | 40 GB to 60 GB        |
+
+Please note that Microsoft Teams requires 3.0 GB of available disk space
```
Comment (JohanFreelancer9) [inline Teams/teams-for-vdi.md:120]:
```suggestion
> [!NOTE]
> Microsoft Teams requires 3.0 GB of available disk space.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @fernandoklurfan, Kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by fernandoklurfan** [comment]
HI All
Did this go live?


-f

**Comment by fernandoklurfan** [comment]
Hi David
Can you help me out with this publishing?

Thanks!

From: Fernando Klurfan
Sent: Monday, March 27, 2023 2:09 PM
To: 'MicrosoftDocs/OfficeDocs-SkypeForBusiness' ***@***.***>; MicrosoftDocs/OfficeDocs-SkypeForBusiness ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [MicrosoftDocs/OfficeDocs-SkypeForBusiness] Update teams-for-vdi.md (PR #9857)

HI All
Did this go live?

-f

**Comment by scanum** [comment]
@dstrome
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by fernandoklurfan** [comment]
Hi All
Can we get an update?
We need this urgently because of customer escalations.


-f

**Pull request merged**